compiler/damlc/daml-compiler/src/DA/Daml/Compiler/DataDependencies.hs

```diff
@@ -355,29 +306,53 @@ isConstraint = \case
         ]
     _ -> False
 
-convType :: Env -> LF.Type -> HsType GhcPs
+genModule :: Env -> LF.PackageRef -> LF.ModuleName -> Gen Module
```

Comment:
I really hate that GHC’s `Module` type is not a module but just a reference to it. This makes the names here a bit confusing since it sounds like we are generating the source for a module. But I don’t really have a less confusing suggestion sadly :disappointed:

Reply:
I agree, `Module` is very counterintuitive name for a module reference.